build/broccoli/strip-glimmer-utilities.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+const babel = require('broccoli-babel-transpiler');
+const stripGlimmerUtils = require('babel-plugin-strip-glimmer-utils');
+const debugMacros = require('babel-plugin-debug-macros').default;
+
+/**
+ * Optimizes out Glimmer utility functions and strips debug code with a set of
+ * Babel plugins.
+ */
+module.exports = function(jsTree) {
+  return babel(jsTree, {
+    annotation: 'Babel - Strip Glimmer Utilities',
+    sourceMaps: 'inline',
+    plugins: [
+      [debugMacros, {
```

Comment:
This is going to need a plugin to strip the `classCallChecks`. Likely you can move this [one I wrote](https://github.com/emberjs/ember.js/blob/master/broccoli/transforms/strip-class-call-check.js) to this guy https://github.com/ember-cli/ember-build-utilities and just use it directly.